components/footer/footer.snip.html

```diff
@@ -24,7 +24,9 @@
       <nav class="ampstart-footer-nav">
         <ul class="list-reset flex flex-wrap mb3">
           {{#links}}
-            <li class="px1"><a class="text-decoration-none caps h5" href="{{{url}}}">{{text}}</a></li>
+            <li class="px1">
```

Comment:
remove newline - i think the whitespace here causes some issue in a browser (just playing safe)